projects/1.12.2/assets/srp-addon-cotesia-glomerata/srpcotesia/patchouli_books/hyperbiology_and_you/zh_cn/categories/blocks.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "方块",
+  "description": "可用于合成、掉落，或与寄生体密切相关的方块。",

```

Blocks that can be synthesized from, dropped by, or are tangentially related to the parasites.
什么方块？
be synthesized from 自（由）……合成的 （diamond can be synthesized from graphite）
dropped by 由谁掉落的
tangentially related 无关地相关 → 并非紧密相关 → 有些相关

- 和谁相关？
- tangentially related to the parasites. 与寄生体有点关系的……

```suggestion
  "description": "自寄生体合成、掉落，或勉强与之有关的方块。",
```
我不确定这里的be synthesized from是用寄生体的尸体/器官制作的还是别的？

---

检查时一直没看到那个tangentially，真是目光呆滞了（←不不忽略指代的问题也很严重好吗）
至于be synthesized from，目录里面有如你所说的，可以拿寄生体组织碎片和别的材料在工作台合成得到的方块，也有寄生体自己产出，无法通过常规手段（尤指利用工作台合成）获取的方块
所以我在想要不要换成“由”这个两种解释都可行的措辞，“由寄生体合成”，或者再准确一点，“由寄生体（或其样本）合成”

*后面item目录有个同一个格式的文本，后续上传改版会一起改掉